crates/control/src/orientation_filter.rs

```diff
@@ -24,16 +33,16 @@ pub struct CreationContext {}
 #[context]
 pub struct CycleContext {
     sensor_data: Input<SensorData, "sensor_data">,
-    cycle_time: Input<CycleTime, "cycle_time">,
-    sole_pressure: Input<SolePressure, "sole_pressure">,
-
-    orientation_filter_parameters: Parameter<Parameters, "orientation_filter">,
+    filter_gain: Parameter<f32, "orientation_filter.filter_gain">,
+    calibration_steady_threshold: Parameter<f32, "orientation_filter.calibration_steady_threshold">,
+    calibration_smoothing_factor: Parameter<f32, "orientation_filter.calibration_smoothing_factor">,
+    num_calibration_cycles: Parameter<usize, "orientation_filter.num_calibration_cycles">,
```

Comment:
please change `num_` to `number_of_` or similar

Reply:
ok